corehq/apps/app_manager/suite_xml/utils.py

```diff
@@ -119,3 +119,17 @@ def get_ordered_case_types(case_type, additional_case_types=None):
     additional_case_types = additional_case_types or []
     additional_types = set(additional_case_types) - {case_type}
     return [case_type] + sorted(additional_types)
+
+
+def is_valid_results_instance_name(app, instance_name):
+    # avoid circular import
+    from corehq.apps.app_manager.suite_xml.post_process.remote_requests import (
+        RESULTS_INSTANCE,
+        RESULTS_INSTANCE_INLINE,
+        RESULTS_INSTANCE_BASE
+    )
+
+    valid_instance_names = {RESULTS_INSTANCE, RESULTS_INSTANCE_INLINE}
+    valid_instance_names.update(f'{RESULTS_INSTANCE_BASE}{module.search_config.instance_name}'
```

Comment:
should this be `get_instance_name()`? If not defined, this will add `results:None` (or possibly `results:` as valid names.

Reply:
Yes thanks for catching that, [e14c0d8](https://github.com/dimagi/commcare-hq/pull/33635/commits/e14c0d8c254ed6b5f94cdf3e6435e376e090b058)